docs/source/templates/readme.rst_t

```diff
@@ -37,6 +37,10 @@ and update with::
 
    conda update {{ name }}
 
+or use the docker container::
+
+   docker pull quay.io/repository/biocontainers/{{ name }}
```

Comment:
We do not have a :latest container, so a user needs to specify a version. 

what about `docker pull quay.io/repository/biocontainers/{{ name }}:<revision>` and maybe even link the revision to the `https://quay.io/repository/biocontainers/{{ name }}?tab=tags`

Reply:
In theory I could pull the latest revision via Quay's API ...

Reply:
But yeah - I guess leaving that to the user is better. You don't have the latest tag intentionally, right?

Reply:
Yes intentionally ...